### PR TITLE
Fix quantization for input to reference model

### PR DESCRIPTION
**Pull request by per**
Add the zerpoint instead of subtracting. This worked since the tests so far used the ones as inputs which quantize to a zp of -128 which gives the same np.int8 result in both cases since the int8 wraps. Also needs to round and clip the scaled values to the int8 range.

Signed-off-by: Per Åstrand <per.astrand@arm.com>

### CODE REVIEW

**Comment by per** [comment]
Will fix the decorators.

File: backends/arm/test/ops/test_add.py [hunk 1]
```diff
@@ -114,42 +126,42 @@ def _test_add_u55_BI_pipeline(
...
+        test_data = (operand1, operand2)
         self._test_add_tosa_BI_pipeline(self.Add2(), test_data)
 
     @unittest.skipIf(
         not VELA_INSTALLED,
         "There is no point in running U55 tests if the Vela tool is not installed",
     )
-    def test_add2_u55_BI(self):
-        test_data = (torch.ones(1, 1, 4, 4), torch.ones(1, 1, 4, 1))
+    @parameterized.expand(Add2.test_parameters)
```
Comment (per) [inline backends/arm/test/ops/test_add.py:164]: @parameterized and @unittest causing failures.

**Comment by digantdesai** [comment]
Is this ready?

File: backends/arm/test/ops/test_add.py (hunk 1 above)
Comment (digantdesai) [reply backends/arm/test/ops/test_add.py]: still an issue?

**Review by digantdesai — APPROVED** [review]
LGTM

**Pull request closed without merging**